src/pages/changelog/post-11.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: "Local Development Support"
+date: "20 Mar 2023"
+layout: "@changelogs"
+description: "Lots of new features and improvements to the Space CLI."
+---
+
+Since the introduction of Deta Space, we have receive a lot of requests for local development support.
```

Comment:
```suggestion
Since the introduction of Deta Space, developers have been asking for a better experience when building locally for Space. Whether it is managing keys or coordinating between two separate [Micros](https://deta.space/docs/en/basics/micros#whats-a-micro), local development requires  unnecessary work.
```